pkg/state/transaction_differ_test.go

```diff
@@ -18,7 +19,8 @@ const (
 	priceConstant = 10e7
 )
 
-var defaultTimestamp = settings.MustMainNetSettings().CheckTempNegativeAfterTime //nolint:gochecknoglobals // no writes
+//nolint:gochecknoglobals // no writes
+var defaultTimestamp = settings.MustMainNetSettings().CheckTempNegativeAfterTime + 1
```

Comment:
Why `+1`?

Reply:
Restored initial value. Short explanation why it should be increased for new test added. Value increased only in new test.